recipes/medcam/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "medcam" %}
+{% set version = "0.1.7" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/medcam-{{ version }}.tar.gz
+  sha256: fc003781c8997271f10be0abc916dc8276eef59bf11ccde307b3ca14a975d84d
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - certifi
+    - cycler
+    - decorator
+    - future
+    - imageio
+    - kiwisolver
+    - mako
+    - markdown
+    - markupsafe
+    - matplotlib-base
+    - networkx
+    - nibabel
+    - numpy
+    - opencv
+    - packaging
+    - pandas
+    - pillow
+    - pyparsing
+    - python >=3.6
+    - python-dateutil
+    - pytz
+    - pywavelets
+    - scikit-image
+    - scipy
+    - simpleitk
+    - six
+
+test:
+  imports:
+    - medcam
+
+about:
+  home: https://pypi.org/project/medcam/
+  summary: An easy to use library that makes model predictions more interpretable for humans. 
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
```

Comment:
Might consider adding `dev_url: https://github.com/MECLabTUDA/M3d-Cam` since that was tricky to find.